ecs.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symplify\EasyCodingStandard\ValueObject\Option;
+use Symplify\EasyCodingStandard\ValueObject\Set\SetList;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->import(SetList::PSR_12);
+    $containerConfigurator->import(SetList::SYMPLIFY);
+    $containerConfigurator->import(SetList::COMMON);
+    $containerConfigurator->import(SetList::CLEAN_CODE);
+
```

Comment:
Can you add the Symfony ruleset please :pray:

Reply:
Sure, I thought has this setup for Symfony related stuff but I will check it 👍  Thanks for your time! I will try this week to split it as requested :)

Reply:
Do you want me to leave the 3 rules?

I've switched to symfony and it made change in `yoda_style`.

Is this what you want?

```diff
-        if (empty($this->innerHits) === false) {
+        if (false === empty($this->innerHits)) {
             $result['inner_hits'] = array_map(fn (InnerHit $hit) => $hit->build(), $this->innerHits);
         }
```

Reply:
Yes it's perfect. I work with yoda condition for so long I can't go back ! 

But in your exemple I can't figure why you don't use a simple `if (!empty($this->innerHits))`. The empty function only return bool so you don't have to be strict.

Reply:
Yeah, in our team we are do like to always use a strict comparison even for bools. We find it more readable than finding `!` symbol at the start